src/Command/SearchPopulateCommand.php

```diff
@@ -62,6 +62,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->populateService->setProgressBar($progressBar);
         $this->populateService->populate($index, $id);
 
+        $output->writeln('');
```

Comment:
What is the purpose of this? Either remove or explain in comment

Reply:
Comment added